overlay-inputs.yml

```diff
@@ -0,0 +1,2 @@
+org_name: {acronym: "CMS", full_form: "Centers for Medicare and Medicaid"}
+disable_slow_controls: true
```

Comment:
We shouldn't be disabling slow controls by default -- we usually wind up doing that when testing, but in real profile runs that should be something people have to do intentionally; please strike this line